invisible_cities/detsim/simulate_electrons_test.py

```diff
@@ -0,0 +1,144 @@
+import numpy  as np
+
+from collections import namedtuple
+from operator    import itemgetter
+
+from pytest import fixture
+
+import invisible_cities.core.system_of_units as units
+
+from invisible_cities.io.mcinfo_io import load_mchits_df
+
+from invisible_cities.cities.detsim_simulate_electrons import generate_ionization_electrons
+from invisible_cities.cities.detsim_simulate_electrons import drift_electrons
+from invisible_cities.cities.detsim_simulate_electrons import diffuse_electrons
```

Comment:
```suggestion
import .. core.system_of_units as units

from ..io.mcinfo_io import load_mchits_df

from . simulate_electrons import generate_ionization_electrons
from . simulate_electrons import drift_electrons
from . simulate_electrons import diffuse_electrons
```
(notice that the `detsim_` prefix is also removed)